cypress/integration/models/stakeholdergroups.ts

```diff
@@ -52,20 +76,26 @@ export class Stakeholdergroups {
             .within(() => {
                 click(editButton);
             });
-        this.getStakeholdergroupName();
-        this.getStakeholdergroupDescription();
+
+        this.stakeholdergroupName = name || this.stakeholdergroupName;
```

Comment:
Why are we doing this ?

Reply:
I am expecting test case to send name to edit and delete. In some cases, we will create more than one stakeholder groups. If we want to delete particular stakeholder then we should send its name.
for example - we created 5 stakeholder groups in  a row then last element will be the value of ` this.stakeholdergroupName` then if we call delete without sending name; it will delete last one. But If I want to delete first one then I should have this argument available to send.